tests/keras_autolog/test_keras_autolog.py

```diff
@@ -205,6 +205,7 @@ def keras_random_data_run_with_callback(
             patience=patience,
             min_delta=99999999,
             restore_best_weights=restore_weights,
+            verbose=1,
```

Comment:
Added this line to trigger the keras tests.

Reply:
Can we keep this change to make it easier to debug issues?

Reply:
Sure - makes sense to keep this!